.editorconfig

```diff
@@ -236,3 +236,86 @@ dotnet_naming_rule.interface_types_must_be_prefixed_with_i.style
 dotnet_naming_style.prefix_private_field_with_underscore.capitalization           = camel_case
 dotnet_naming_style.prefix_private_field_with_underscore.required_prefix          = _
 
+# Code files
+[*.{cs,vb}]
+
+dotnet_diagnostic.CA1001.severity = warning
```

Comment:
Are all of these lines where we differ from the standard? Or are the just all listed?

Wondering if we want any comments here around our choices for posterity?

Reply:
I just ported what we had. Only the last 3 rules are new, since they were a change from 1.0.x to 1.1.x.

Reply:
I don't know the explanations for the other rules, but I can add them for the new ones if you want.

Reply:
No this is perfect for now, thanks! We can open up another issue later about auditing any other StyleCop specific rules we have.